docs/releases/3.0.txt

```diff
@@ -168,6 +168,9 @@ Management Commands
 * :option:`showmigrations --list` now shows the applied datetimes when
   ``--verbosity`` is 2 and above.
 
+* On PostgreSQL, :djadmin:`dbshell` now supports client-side TLS certificates
+  and key (:ticket:`30370`).
```

Comment:
As this is not a patch release, we do not include ticket references in the release notes.

Reply:
All you comments fixed